.github/docker/Dockerfile.alpine:3.16

```diff
@@ -1,11 +1,11 @@
-# Example of how to build this container (ran from pgx checkout directory):
-#   docker build --build-arg PG_MAJOR_VER=14 -f .github/docker/Dockerfile.alpine\:3.16 -t pgx-alpine .
+# Example of how to build this container (ran from pgrx checkout directory):
+#   docker build --build-arg PG_MAJOR_VER=14 -f .github/docker/Dockerfile.alpine\:3.16 -t pgrx-alpine .
 #
 # Example of how to run this container with tests after building the image:
-#   docker run pgx-alpine cargo test --no-default-features --features pg14 --locked
+#   docker run pgrx-alpine cargo test --no-default-features --features pg14 --locked
 #
 # Note that "PG_MAJOR_VER" build arg in the build step must match the
-# "--features pgXX" in the run step
+# "--features pgrxX" in the run step
```

Comment:
This (and all the other instances of `pgXX`) shouldn't have been changed

Reply:
nice catch.  thanks.  we're still reviewing this too.